tests/actions.go

```diff
@@ -376,6 +376,15 @@ func (oa *operatorActions) DeployOperator(info *OperatorConfig) error {
 		return fmt.Errorf("failed to deploy operator: %v, %s", err, string(res))
 	}
 
+	// delete statefulset update webhook and configuration
+	cmd = fmt.Sprintf("kubectl delete -f %s/webhook.yaml", oa.manifestPath(info.Tag))
```

Comment:
why move the cleaner to `DeployOperator`?

Reply:
manifests dir will only exist after checkouttag in stability test